modules/api/README.md

```diff
@@ -0,0 +1,50 @@
+# API
+
+## Purpose
+
+The api module is intended to provide a stable, versioned REST interface
+for interacting with the core LORIS functionality without requiring backend
+SQL access.
+
+## Intended Users
+
+The module has 2 intended use cases:
+1. Users (or creators) of LORIS data that are external to the LORIS core
+   software that must extract or import data from LORIS.
+2. Internal LORIS modules which retrieve data from the API.
+
+## Scope
+
+The API only provides information about concepts that are cross-module
+or data that is of general concern to users. 
```

Comment:
what does `general concern` actually mean-- that the super-technical metadata is not available?  
It would be nice to state this more clearly as "all/most data visible in the web front-end is accessible via API"

Reply:
It means it's a cross module concern that multiple people would be interested in, not something added because a single module needs it. It's a little vague because it's not very well defined what should be in the API and what should be in a module at the moment